google/ggrc-core
# GGRC-5519 Add type attribute to buttons

# Issue description

Buttons that don't have `type` specified, have default `type="submit"` which can lead to unexpected submits of forms.

# Steps to test the changes

Check that buttons without specified event handler still work. Check some buttons with provided click handlers.

# Solution description

Add `type="button"` to all buttons without type.

# Sanity checklist




# PR Review checklist
